File: sdks/thunkmetrc-cs/ThunkMetrc.Wrapper/Models/PlantBatchesCreateAdjustPlantBatchesRequest.cs
using System;
using System.Collections.Generic;

namespace ThunkMetrc.Wrapper.Models
{
    public class PlantBatchesCreateAdjustPlantBatchesRequest
    {
        [global::System.Text.Json.Serialization.JsonPropertyName("AdjustmentDate")]
        public string? AdjustmentDate { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("AdjustmentReason")]
        public string? AdjustmentReason { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("PlantBatchName")]
        public string? PlantBatchName { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("Quantity")]
        public int? Quantity { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("ReasonNote")]
        public string? ReasonNote { get; set; }
    }
}

File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/PlantBatchesCreateAdjustPlantBatchesRequest.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class PlantBatchesCreateAdjustPlantBatchesRequest {
    @JsonProperty("AdjustmentDate")
    public String adjustmentDate;
    @JsonProperty("AdjustmentReason")
    public String adjustmentReason;
    @JsonProperty("PlantBatchName")
    public String plantBatchName;
    @JsonProperty("Quantity")
    public Integer quantity;
    @JsonProperty("ReasonNote")
    public String reasonNote;
}
